template/src/components/Button.tsx

```diff
@@ -9,17 +9,17 @@ import {
   LayoutProps,
 } from 'styled-system';
 
-import { Container } from '../Container';
-import { Text } from '../Text';
-import { Touchable } from '../Touchable';
-import { colors } from '../../styles';
-import { AccessbilityRole } from '../../types/AccessibilityRole';
+import { AccessbilityRole } from '../types/AccessibilityRole';
+import { colors } from '../styles';
+import { Container } from './Container';
+import { Text } from './Text';
+import { Touchable } from './Touchable';
```

Comment:
If we're going to sort these now, let's make sure to add `import/order` plugin to our react-native eslint plugin (I've created a followup issue here: https://github.com/echobind/eslint-plugin-echobind/issues/141)